sqlx-core/src/sqlite/options/connect.rs

```diff
@@ -50,7 +49,7 @@ impl SqliteConnectOptions {
 
         for (key, opt_value) in &self.pragmas {
             if let Some(value) = opt_value {
-                write!(string, "PRAGMA {} = {}; ", key, value).ok();
+                string.push_str(&format!("PRAGMA {} = {}; ", key, value));
```

Comment:
You should only use `.push_str()` if you already have just a `&str` to append to the string. In this case `write!()` is preferable.

Reply:
Sorry for the misuse.